diktat-ruleset/src/test/kotlin/com/saveourtool/diktat/ruleset/smoke/DiktatSaveSmokeTest.kt

```diff
@@ -183,20 +178,5 @@ class DiktatSaveSmokeTest : DiktatSmokeTestBase() {
                 diktatFrom?.copyTo(diktat, overwrite = true)
             }
         }
-
-        @AfterAll
-        @JvmStatic
-        internal fun afterAll() {
```

Comment:
all files are located in `@TempDir` and junit cleanups them.
I removed it because it cleans the folder regardless of flag in `@TempDir` to keep files (it was required me for debugging)